game.js

```diff
@@ -86,10 +124,11 @@ const clicks = {
 		});
 		$('.tile').on('contextmenu', function(e){
 		return false;
-		});
+	}); // WTF??
 		$('.tile').on('contextmenu', function(event) {
 			let tile = getTileByEvent(event);
 			if (tile.activated && (tile.sumNeighbors('flagged') === tile.sumNeighbors('hasMine'))) {
+				// why do we call this again? already been called in click event
```

Comment:
you can use left click or right click to expand around a tile that already has all its mines flagged. I did that cause left-clicking on a mac is kind of a pain. So I copy-pasted it into both event listeners. Might be cleaner to abstract it into a method or something, but it only occurs twice.